app/views/content/life-as-a-teacher/change-careers.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Career change to teaching"
```

Comment:
This is what is being displayed on the page where I presume you wanted the h1 to appear?

Might be worth checking with Spencer/Jen about what is displayed/what should be?